packages/synapse-react-client/src/testutils/muiDatePickerMock.tsx

```diff
@@ -1,7 +1,11 @@
 // Mock the DatePicker component from MUI
 vi.mock('@mui/x-date-pickers', () => ({
   DatePicker: ({ value, onChange, slotProps }: any) => {
-    const textFieldProps = slotProps?.textField ? slotProps.textField({}) : {}
+    let textFieldProps = slotProps?.textField
+    if (typeof slotProps?.textField === 'function') {
+      // If textFieldProps is a function, call it to get the props
+      textFieldProps = slotProps?.textField({ value, onChange })
+    }
```

Comment:
Fix a bug in the mock (I think previously slotProps?.textField was always a function, but it may not be)